thunder/tests/test_grad.py

```diff
@@ -281,7 +281,7 @@ def check_vjp(f, *primals, comp, executor="torch", set_compile_data: bool = Fals
     """
     # Let f be a function from vectors of size n to vectors of size m.
     # Its Jacobian is a matrix J of size m x n.
-    # The adjoint property is J^* J = I, where J^* is the conjugate transpose (adjoint) of J.
+    # Represent by J^* the conjugate transpose (adjoint) of J.
```

Comment:
@IvanYashchuk to review these changes

Reply:
The problem with this comment is that this is actually the unitary property, that is, a linear transformation is unitary iff its inverse is also its adjoint.  For a linear operator J and a bilinear form <*, *>, the adjoint of J, denoted J^*, is the linear operator satisfying: for all vectors u and v, <Ju, v> = <u, J^*v>.  In R^n with the usual inner product, this is equivalent to J^* being the conjugate transpose of J.  It's a relatively rare occurrence that this J^* also happens to be J^-1, and in fact, J^* exists even when J is not (left- or right-) invertible.  For real-valued square matrices, the unitary transformations are exactly the orthogonal (rotation) matrices, whereas it's possible to take the transpose of any real matrix to find its adjoint.